packages/cli/tsconfig.json

```diff
@@ -19,7 +19,13 @@
 		"target": "es2017",
 		"sourceMap": true,
 		"emitDecoratorMetadata": true,
-		"experimentalDecorators": true
+		"experimentalDecorators": true,
+		"baseUrl": ".",
```

Comment:
ist this supposed to be here?

Reply:
As discussed, we're keeping it here, but there is an alternative without `baseUrl`.